### ISSUE TITLE
Weird results with sqrt

### ISSUE THREAD

**Issue by delwarde** [jiggzson/nerdamer#36]
Entering formulas like 
```
sqrt(2x)
```
or
```
sqrt(2*x)
```
etc.  give weird results like  
 ```
√((3880899 x)/2744210)
```

**Comment by delwarde** [comment]
Another bug with sqrt :
```
2*sqrt(x)
```
render as
```
sqrt(2x)
```

**Comment by jiggzson** [comment]
This issue seems to be related to the rendering of the math. As @brosnanyuen suggested we'll be switching to Ascii2Math in future versions. I'll look into this issue. Thanks for pointing it out @delwarde

**Comment by jiggzson** [comment]
Unfortunately sqrt(2x) will still produce something like 3880899√ x/2744210 because nerdamer evaluates sqrt(2) right away which in turn gets converted to a fraction. In future versions I hope to avoid this when addressing issue #8 and additionally try to keep numbers in the form sqrt(n) for as long as possible.

**Comment by jiggzson** [comment]
Addressed in version 0.6.0. Evaluation of sqrt is deferred until evaluate is called.
